Models.py

```diff
@@ -27,6 +27,9 @@ class NotamRequest:
     def __init__(self, data):
         self.startAirport = data.get('startAirport', None)
         self.destAirport = data.get('destAirport', None)
+        self.additionalAirport1 = data.get('destinationLocation2', None)
+        self.additionalAirport2 = data.get('destinationLocation3', None)
+        self.additionalAirport3 = data.get('destinationLocation4', None)
```

Comment:
so we explicitly support only 4 destinations?  why not define a loop and list for these?
```python
self.destinations = [ data.get('destAirport', None) ]
index = 2
while data.get( f'destinationLocation{index}', None ) is not None:
    self.destinations.append( data.get(f'destinationLocation{index}') )
    index += 1
```

Reply:
Good Point Sir, I'll make a change on that

Reply:
I agree we probably want a loop. You could change the model to hold a list of points rather than just the start and end